packages/vite/src/node/cli.ts

```diff
@@ -125,6 +148,7 @@ cli
       )
 
       server.printUrls()
+      stopProfiler((message) => server.config.logger.info(`  ${message}`))
```

Comment:
Should this be moved after the `catch` similar to the build command?

Reply:
Finally is good for me 👍